.gitignore

```diff
@@ -9,3 +9,5 @@ docs/build/
 **/.DS_Store
 
 **/nitro-testnode
+
+**/proptest-regressions
```

Comment:
We shouldn't ignore these! It makes failure cases load faster the next time tests run 👀

Reply:
make sense